README.md

```diff
@@ -188,3 +188,71 @@ for me that is https://wapp-to-do-mod9-jl.azurewebsites.net/
 
 
 
+Add a Store for the State in azurre
+
+````powershell
+az login --use-device-code
+
+$RESOURCE_GROUP_NAME='Cohort22_JonLon_ProjectExercise'
+$STORAGE_ACCOUNT_NAME="tfstate$(Get-Random)"
+$CONTAINER_NAME='tfstate'
+
+# Create resource group if group dosnt exsit
+#New-AzResourceGroup -Name $RESOURCE_GROUP_NAME -Location UKsouth
+
+# Create storage account
+$storageAccount = New-AzStorageAccount -ResourceGroupName $RESOURCE_GROUP_NAME -Name $STORAGE_ACCOUNT_NAME -SkuName Standard_LRS -Location UKsouth -AllowBlobPublicAccess $false
+
+# Create blob container
+New-AzStorageContainer -Name $CONTAINER_NAME -Context $storageAccount.context
+
+$TERRAFORM_STATE_KEY=(Get-AzStorageAccountKey -ResourceGroupName $RESOURCE_GROUP_NAME -Name $STORAGE_ACCOUNT_NAME)[0].value
+$env:ARM_ACCESS_KEY=$TERRAFORM_STATE_KEY
+
+````
+
+To run on the pipeline we need some secrests setting up
+
+update the following with your required values
+$RGName = '<Resource Group Name>'
+$SubScriptionID = "<Subscription ID>"
+$WebAppName = '<Web App Name>'
+$ServicePrincipleName = "<Service Principle Name>"
+
+````powershell
+
+$RGName = 'Cohort22_JonLon_ProjectExercise'
+$SubScriptionID = "d33b95c7-af3c-4247-9661-aa96d47fccc0"
+$WebAppName = 'Wapp-To-Do-Mod9-JL'
+$ServicePrincipleName = "Dev-SP-User"
+
+az login
+az account set --subscription=$SubScriptionID
+$spJson = az ad sp create-for-rbac --name $ServicePrincipleName --role Contributor --scopes /subscriptions/$SubScriptionID/resourceGroups/$RGName
+$spDetails = $spJson | convertFrom-Json
+$spJson=''
+
+#Azure Login
+az webapp config appsettings set -g $RGName -n $WebAppName --settings ARM_CLIENT_SECRET=$spDetails.password
+az webapp config appsettings set -g $RGName -n $WebAppName --settings ARM_SUBSCRIPTION_ID=$SubScriptionID
+az webapp config appsettings set -g $RGName -n $WebAppName --settings ARM_TENANT_ID=$spDetails.tenant
+az webapp config appsettings set -g $RGName -n $WebAppName --settings ARM_CLIENT_ID=$spDetails.appID
+$spDetails=''
+
+````
+
+To run locally run the following replacing the relevent data in <>
+
+Add the following values to the .env file 
+ARM_CLIENT_SECRET=<Service principle Password>
+ARM_SUBSCRIPTION_ID=<Subscription ID>
+ARM_TENANT_ID=<Tenant ID>
+ARM_CLIENT_ID=<Service principle ID>
```

Comment:
Similar to the above, this won't actually achieve anything! Locally you don't need this as you're already logged into the `az` command line tool (and if you aren't, I _believe_ that this won't work, since it won't pick up the `.env` file automatically)